README.md

```diff
@@ -1,22 +1,13 @@
 # SwiftSyntax
 
-SwiftSyntax is a set of Swift bindings for the
-[libSyntax](https://github.com/apple/swift/tree/main/lib/Syntax) library. It
-allows Swift tools to parse, inspect, generate, and transform Swift source
-code.
-
-Its API is designed for performance-critical applications. It uses value types almost exclusively and aims to avoid existential conversions where possible.
-
-> Note: SwiftSyntax is still in development, and the API is not guaranteed to
-> be stable. It's subject to change without warning.
-
```

Comment:
I think we should continue to include this in the main README because it gives a high-level overview of what the library does (or at least the best that we currently have).

Reply:
Moved this part, and the part above back to the main readme.